x-pack/plugins/ml/public/embeddables/anomaly_swimlane/embeddable_swim_lane_container.tsx

```diff
@@ -107,6 +107,8 @@ export const EmbeddableSwimLaneContainer: FC<ExplorerSwimlaneContainerProps> = (
     [swimlaneData, perPage, fromPage, setSelectedCells]
   );
 
+  onRenderComplete();
```

Comment:
I'm not sure if it's the right way to report render completion 🤔 
I'll create a PR to utilize the [onRenderChange](https://elastic.github.io/elastic-charts/?path=/story/interactions--render-change-action&globals=toggles.showHeader:true;toggles.showChartTitle:false;toggles.showChartDescription:false;toggles.showChartBoundary:false;theme:light) callback provided by elastic charts.